cypress/integration/userCanEditRecipe.spec.js

```diff
@@ -0,0 +1,61 @@
+describe('User can edit recipe', () => {
+
+  beforeEach(() => {
+    cy.route({
+      method: 'GET',
+      url: 'http://localhost:3000/v1/recipes/1',
+      response: 'fixture:single_recipe.json',
+      status: 200
+    })
+  })
+
+  it('Successfully', () => {
+    cy.route({
+      method: 'PUT',
+      url: 'http://localhost:3000/v1/recipes/1',
+      status: 201,
+      response: '{ "message": "Your recipe has been updated." }'
+    })
+
+    cy.get('#recipe-1').click({ force: true })
+
+    cy.get('[name="edit-recipe"]').click()
+
```

Comment:
Please remove spaces.

Reply:
I think this makes it more readable, please have a discussion with the team about this. 
If you all feel the same way, then go ahead and remove them.